crates/downloader/src/main.rs

```diff
@@ -127,6 +127,10 @@ struct Args {
     #[arg(value_enum, long, default_value(Os::default_opt().map(<&str>::from)))]
     os: Os,
 
+    /// ダウンロードする場合のリトライ回数
+    #[arg(long, default_value("5"))]
```

Comment:
wgetを真似るなら`-t`を受け付けるようにしてもいいかもです。

```suggestion
    #[arg(short, long, default_value("5"))]
```